client/src/containers/NewsOfTheDays/component/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Loader from 'react-loader-spinner';
+import './style.scss';
+import { fetchData } from '../action';
+import { connect } from 'react-redux';
+
+
+class NewsOfTheDays extends Component {
+  //Create pass/link to news page to use this function
```

Comment:
I don't like comments. What can we do with that?

Reply:
i must be url for articles we need to create task,
 because now i dont even have the news component